Directory.Build.props

```diff
@@ -33,8 +33,8 @@
     <!-- Embed source files that are not tracked by the source control manager in the PDB. -->
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
 
-    <!-- Working around https://github.com/NuGet/Home/issues/8467 -->
-    <NoWarn>$(NoWarn);NU5131</NoWarn>
+    <!-- Working around https://github.com/dotnet/sdk/issues/24747 -->
+    <NoWarn>$(NoWarn);NU1505</NoWarn>
```

Comment:
I feel like this is a good example: This threw when the build was run in some ways locally and on the CI, but not others. That's exactly the kind of confusion I'd like to avoid.